main/src/ca/uwaterloo/flix/language/phase/typer/PrimitiveEffects.Methods.json

```diff
@@ -1,11 +1,20 @@
 {
   "methods": {
     "java.lang.Boolean::getBoolean": "Env, IO",
-
+    "java.lang.Runtime::getRuntime": "IO",
+    "java.lang.Runtime::exit": "Sys, IO",
+    "java.lang.Runtime::addShutdownHook": "Exec, Sys, IO",
+    "java.lang.Runtime::removeShutdownHook": "Sys, IO",
+    "java.lang.Runtime::halt": "Sys, IO",
```

Comment:
Should we have something more dangerous than `Sys`? The documentation says that `halt` should be used with extreme care and halts the JVM unconditionally which may lead to data corruption.

Reply:
I dont think halt is worse than running processes or load arbitrary machine code.